### PR TITLE
My first code review at Microverse.

### PR DESCRIPTION
**Pull request by lawrahkonwea**
This simple landing page was built using;
- Html
- Cascading style sheet (CSS)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                <input type="text" name="Email" class="form-input" placeholder="Email" required />
+            </div>
+
+            <div class="form-register">
+                <label for=" password"></label>
+                <input type="password" name="Password" class="form-input" placeholder="Password" required />
+            </div>
+
+            <div class="form-register"><button type="button" class="reg-button" type="btn">CLAIM YOUR FREE TRAIL</button></div>
+            <div class="terms ">
```
Comment (lawrahkonwea) [inline index.html:46]: Kindly wrap the form element in a <div> tag, so the page can be responsive.

**Review by lawrahkonwea — COMMENTED** [review]
# Hi @lawrahkonwea,
I'm Lawrah your reviewer for this PR. 👋 👋

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# Highlights
### ✔️ Good PR summary.
### ✔️ Followed GitHub flow.
### ✔️ No linter errors.

# Required Changes ♻️
Check the comments under the review.

Check ReadMe file and make it more descriptive.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @lawrahkonwea  ,

Nice job :clap: :100: , you are almost there.



## (Highlights) Good Points: :+1:

- :heavy_check_mark: Descriptive PR title & description.
- :heavy_check_mark: Inline comment is clear and easy to understand.

## Required Changes :recycle: 

 - [ ] Your review should have at least two comments. So far you have just one comment. Please add one more comment as requested in the [project requirements ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md#project-requirements)
 > requirement
## [Optional] suggestions:
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (lawrahkonwea) [inline index.html:8]: Optional suggestion.
Kindly add a descriptive title to your website.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+        <form action="" class="formation">
+
+            <div class="form-register">
+                <label for="First Name"></label>
+                <input type="text" name="First Name" class="form-input" placeholder="First Name" required />
+            </div>
+
+            <div class="form-register">
+                <label for="Last Name"></label>
```
Comment (lawrahkonwea) [inline index.html:31]: Kindly wrap the form element in a

tag, so the page can be responsive.

**Review by lawrahkonwea — COMMENTED** [review]
# Hi @lawrahkonwea,
I'm Lawrah your reviewer for this PR. 👋 👋

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights
✔️ Good PR summary.
✔️ Followed GitHub flow.
✔️ No linter errors.
Required Changes ♻️
Check the comments under the review.

Also, please check ReadMe file and make it more descriptive.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by sajid-munawar — APPROVED** [review]
### Hi @lawrahkonwea ,
Great job on this project review! 👍🏻
# Status: Approved ✔️
Excellent job on this project review. Keep it up 👏
Your project review is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

### To Highlight
- [x]  Added descriptive inline comments. :heavy_check_mark: 
- [x]  Review comments are polite, use suggestions and give reasons for requested change :heavy_check_mark:
- [x] Made use of emojis in project review summary to make the code review more human-friendly 👽🤖. :heavy_check_mark: 
- [x] Made use of [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) :heavy_check_mark: 

 ## [Optional] suggestions:
 * Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**